docs/what-is-regression-games.mdx

```diff
@@ -2,58 +2,57 @@
+- **Cutting-Edge Technology:** We blend traditional algorithms with state-of-the-art machine learning, creating a powerful synergy that tackles the unique challenges of game testing.
+- **Developer-Centric Approach:** Our tools are built by developers, for developers. We understand your pain points and have crafted solutions that seamlessly integrate into your workflow.
 
-This product is currently a **release preview**. We are actively talking with studios and game developers to discover the best features and use cases to tackle, and we're making changes frequently.
+## What is our vision?
 
-The team has focused on the foundation of supporting use cases and initial infrastructure to get bots to connect to Unity. The bulk
-of our value will come from the interfaces to build bots, the scale of our infrastructure, the speed of integration, and the products
-built around these bots.
+It's not secret that development teams are hesitant to adopt automated testing. If not done correctly, tests can become a burden rather than a way
+to supercharge your development process.
 
-![timeline feature](unused-content/session-insights/img/replay/timeline.png)
+Our vision is to build tools with the community that actually help you be more productive. There are four core principles driving the development of this framework:
+- **Ease of use** - Developers should be able to integrate and create new tests with minimal effort.
+- **Reliability** - In the dynamic world of games, reliability is key. Our framework is designed to handle the unexpected and give you the tools to create robust, dependable tests.
+- **Data driven** - Our framework collects comprehensive data, empowering you to build smarter automations and create more effective tests.
 
-## Availability Assurances
+Automated testing of games is challenging due to the complexity and dynamic behavior of games. This is why we
+provide a variety of approaches to create **bots** that can handle the unique constraints of your game. We aren't locked into either old school
+AI approaches or LLMs, but rather utilize any technique when it will actually result in a bot that can test your game reliably.
 
-The Regression Games team offers the following assurances for our initial testing with developers. Additionally, we have a dedicated Discord server for support, and can also create a Slack Connect channel, giving you direct access to our entire team. For access to
-these channels, send an e-mail to [aaron@regression.gg](mailto:aaron@regression.gg).
+## Highlighted Features
 
-* A single account can run at least 10 bots concurrently
-* The time from requesting a bot to be started to the time it connects to Unity will take less than 10 seconds if the bot is relying on dependencies that are common to most bots. This does not cover bots that may have lots of library dependencies, which may take longer.
+The table below highlights some of the main features of the framework.
```

Comment:
Imo you don't need a primer statement if the entire section _is_ this table. It'll be clear to readers what the contents of the table are.

Reply:
Removed